torch/fx/experimental/proxy_tensor.py

```diff
@@ -183,23 +183,30 @@ def fetch_tensor_proxy(tracer):
     return lambda t: get_proxy_slot(t, tracer, t)
 
 
-def proxy_call(proxy_mode, func_overload, args, kwargs=None):
+def proxy_call(proxy_mode, func, args, kwargs=None):
     if kwargs is None:
         kwargs = {}
 
-    func = func_overload.overloadpacket
-    if func_overload in CURRENT_DECOMPOSITION_TABLE:
+    def can_handle_tensor(x):
+        return type(x) in HANDLED_TYPES or has_proxy_slot(x, proxy_mode.tracer)
```

Comment:
if the tensor has a proxy slot, then it must be in HANDLED_TYPES?

Reply:
but it can have a proxy slot and not be in `HANDLED_TYPES` :^)

e.g. FakeTensor or other tensor subclasses.